CMakeLists.txt

```diff
@@ -359,7 +360,7 @@ if(BUILD_PIPS_IPM OR BUILD_PIPS_NLP)
     # - scalapack and blacs should be set in ${SCALAPACK_LIBRARIES} using Toolchain.cmake (or hardcoded here)
     # - mpifort also needed
     # - parmetis 5.0 or later required by mumps but no need to specify it here: it should be detected by the build system at this point
-    set(MUMPS_LIBRARY "-L${MUMPS_DIR}/lib -L/home/petra1/work/installs/parmetis-4.0.3/lib -ldmumps -lmumps_common -lpord -lmpifort -lparmetis ${SCALAPACK_LIBRARIES}")
+    set(MUMPS_LIBRARY "-L${MUMPS_DIR}/lib -ldmumps -lmumps_common -lpord -lscalapack-openmpi -lblacs-openmpi  -lblacsF77init-openmpi -lblacsCinit-openmpi -lmpi_mpifh")
```

Comment:
I would test here:
if ${SCALAPCK_LIBRARIES} is empty or not defined, then do your hard-coded set(...)
otherwise (i.e., ${SCALAPACK_LIBRARIES} defined in the Toolchain file) do the set(...) from the master, namely
set(MUMPS_LIBRARY "-L${MUMPS_DIR}/lib -ldmumps -lmumps_common -lpord -lmpifort ${SCALAPACK_LIBRARIES}")

this way we are consistent with the explanation above.